### DISCUSSION TITLE
There is no way to disconnect when on some pages

### DISCUSSION THREAD

**Discussion by jonas-martinez** [lenra-io/lenra_cli#107] [General]
(BACKOFFICE) When in the process of validating your account (registration + dev validation) there is no way to disconnect from your account easily (cleaning cookies works but not practical). This can be quite frustrating when you already have a validated account and want to switch to it.

(APP) There is also no way to disconnect when connected on the app part of Lenra.

**Comment by taorepoara** [comment]
Have been fixed on many pages. Open an issue if find a page where there is no way to disconnect (or go back to a page with a disconnect button).
